howtos/documentation-workflows.MD

```diff
@@ -0,0 +1,44 @@
+---
+title: "Documentation workflows"
+weight: 1
+aliases: [ "/documentation-workflows" ]
+description:
+  A guide to walk you through the Camunda Cloud documentation workflow when working with the
+  GitHub web-based editor. 
+---
```

Comment:
Since this is a plain markdown page I don't think we need this section anymore.

Reply:
@akeller Awesome, removed that section! :)